packages/microsoft-reactnative-sampleapps/windows/SampleLibraryCPP/MyModule.h

```diff
@@ -1,87 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-
-#pragma once
-
-#include "pch.h"
-
-#include <functional>
-#include <sstream>
-
-#include "NativeModules.h"
-
-namespace SampleLibraryCPP {
-
-REACT_MODULE(MyModule);
-struct MyModule {
-  REACT_CONSTANT(m_fieldConst);
-  const int m_fieldConst = 42;
-
-  REACT_CONSTANT(m_fldConst, "fldConst");
-  const int m_fldConst = 43;
-
-  REACT_CONSTANT_PROVIDER(SimpleConstants);
```

Comment:
>REACT_CONSTANT_PROVIDER [](start = 2, length = 23)

Could you please add to the example how we can provide constants using a method?

Reply:
Done. Adding sync examples too.